internal/MozziGuts_impl_TEENSY.hpp

```diff
@@ -42,14 +45,16 @@ void setupFastAnalogRead(int8_t speed) {
 #endif
 }
 
+} // namespace MozziPrivate
 void adc0_isr(void)
 {
-  advanceADCStep();
+  MozziPrivate::advanceADCStep();
 }
+namespace MozziPrivate {
 
 void setupMozziADC(int8_t speed) {
   adc = new ADC();
-  adc->adc0->enableInterrupts(adc0_isr);
+  adc->adc0->enableInterrupts(adc0_isr);  // TODO: is this even correct? Does the function take a parameter? And is adc0_isr a predefined name, or are we free to move this to MozziPrivate?
```

Comment:
Not sure if I understood your comment correctly, but `adc0_isr` is not a predefined name, we define this function just above. Knowing if the isr can be moved in  `MozziPrivate`, I don't see why not.

Reply:
Or maybe not, now seeing your comment on the template.